docs/tutorials/widgets/using-react-in-a-widget.md

```diff
@@ -199,26 +54,22 @@ Nothing warms the heart of a developer like a good "Hello world!" But you probab
 To keep some order in the project, you will put [CKEditor classes](#ckeditor-classes) in the `/ckeditor` directory and [React components](#react-components) in the `/react` directory. [Images and CSS styles](#styles-and-assets) will land in the `/assets` directory. By the time you are finished with this tutorial, the structure of the project should look as follows:
 
 ```bash
-├── app.js
+├── main.js
 ├── assets
-│   ├── product1.jpg
-│   ├── product2.jpg
-│   ├── product3.jpg
-│   ├── product4.jpg
+│   ├── fields.jpg
+│   ├── malta.jpg
+│   ├── tajmahal.jpg
+│   ├── umbrellas.jpg
 │   └── styles.css
 ├── ckeditor
 │   ├── insertproductpreviewcommand.js
 │   └── productpreviewediting.js
-├── dist
-│   ├── bundle.js
-│   └── bundle.js.map
 ├── index.html
 ├── package.json
 ├── react
-│   ├── productlist.js
-│   └── productpreview.js
-├── node_modules
-└── webpack.config.js
+│   ├── productlist.jsx
+│   └── productpreview.jsx
+└── node_modules
```

Comment:
If possible, I would simplify this, and just land everything ( `main.js` and other folders) directly under `src` folder, as this is a more common pattern if you have larger project.

In Vite setup `public` is usually used as a folder for static assets.